views/components/application-card/macro.njk

```diff
@@ -58,7 +69,9 @@
                 <a href="{{ localify(props.editUrl) }}"
                    class="btn btn--small">
                     {{ cardCopy.continue }}
-                    <span class="u-visually-hidden">({{ projectName | safe }})</span>
+                    <span class="u-visually-hidden">
+                        ({{ props.projectName or props.untitledName }})
```

Comment:
I think the initial a11y report was pointing out that a list of untitled applications will always read out "Untitled proposal" at this point, so could we add the datestring into this label?

Reply:
Good shout, I did wonder that as I was updating this.